### ISSUE TITLE
Fix `declaration-property-value-no-unknown` false positives for `-webkit-match-parent`

### ISSUE THREAD

**Issue by HolgerJeromin** [stylelint/stylelint#8605]
### What minimal example or steps are needed to reproduce the bug?

```css
div {
	text-align: -webkit-match-parent;
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": [
    "declaration-property-value-no-unknown": true
  ]
}
```

### How did you run Stylelint?

https://stylelint.io/demo/#N4Igxg9gJgpiBcIoEsBuACYAdAdlgLvjAB74C0AhgDbIDmO8ZA7jAEYDWy5AthfmAAsyABwoAnGDny4AviAA0IAGbIqMAHIVucRCS3C1AOjABnEwvAQcK2ghDYc6dFhAkiOKCZfx0AbVxOTi4m+ACeajRSZJDWdGQhFB7iUC4B6AC6shYxNgBiEGK8+HYAViZWFrDC5ogOQSAh4TCR0gjOIFR8MCEu8mnBYRHIUTlxCUliKW0unUQ9IFkyQA

### Which Stylelint-related dependencies are you using?

```json
{
        "stylelint": "16.20.0",
        "stylelint-config-standard": "38.0.0",
}
```

### What did you expect to happen?

`text-align: -webkit-match-parent` should be valid because blink does not support the unprefixed name.
ref
https://issues.chromium.org/issues/40787216
https://developer.mozilla.org/en-US/docs/Web/CSS/text-align#browser_compatibility

### What actually happened?

It was reported:

> Unexpected unknown value "-webkit-match-parent" for property "text-align"

**Comment by ybiquitous** [comment]
@HolgerJeromin Thanks for the report. I'm not sure how `-webkit-match-parent` is used, but we can work around this issue by specifying [`propertiesSyntax`](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown/#propertiessyntax) like this:

```json
{
  "rules": {
    "declaration-property-value-no-unknown": [true, {
      "propertiesSyntax": {
        "text-align": "| -webkit-match-parent"
      }
    }]
  }
}
```

See also [this demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIoEsBuACYAdAdlgLvjAB74C0AhgDbIDmO86ZA7jAEYDWy5AthfmAAWZAA4UATjBz4A3LgJFSlGvUZ8BwsZOlycAXxAAaEADNkVGADkKPOIhI2RFgHRgAzm6PgIOM7QQg2Djo6Fgg4gCuFm5hjEEhIWGwYFQS-Mg+ouIQIjDi+ACeZKjUETBkOBBkETgclcx4COgA2viRMIaYuAkJYSLZufnIMG4AygXSFMSxXcE9vSCK5NR0jYxhAD5MrJzcZOpCohJS+GHd83rnIXoAuueX+l6QvnQAYhDi6gEAVm4+XrARJ5EPFQiA3IULDRpDMwgBGABszgATAAGZyos6PPRAA).

At this point, I lean towards avoiding the support of vendor prefixes in this rule, but I'd like to hear other ideas.

**Comment by jeddy3** [comment]
It looks like support [recently landed in Blink](https://chromium-review.googlesource.com/c/chromium/src/+/6721711).

Let's close the issue in favour of [locally extending the syntax](https://github.com/stylelint/stylelint/issues/8605#issuecomment-2938834399).

CSSTree may also choose to patch the syntax upstream.
